src/main.ts

```diff
@@ -251,8 +269,33 @@ export default class NewZettel extends Plugin {
     } else {
       titleContent = "";
     }
-    const fullContent = titleContent + content;
-    const file = await this.app.vault.create(path, fullContent);
+
+    let file = null;
+    const backlinkRegex = /{{link}}/g;
+    const titleRegex = /{{title}}/g;
+    if (this.settings.templateFile && this.settings.templateFile != "") {
+      let template_content = "";
+      try {
+        template_content = await this.app.vault.adapter.read(this.settings.templateFile);
+      } catch (err) {
+        new Notice(`Couldn't read template file ${this.settings.templateFile} make sure that it is a valid file...`);
```

Comment:
Here it will check if the template file path is valid and throw an error if not readable (aka. wrong path, no file present, other problems while reading)

Reply:
@Eforen